third_party/rust/keccak/v0_1/BUILD.gn

```diff
@@ -33,4 +33,14 @@ cargo_crate("lib") {
   executable_configs += [ "//build/config/compiler:no_chromium_code" ]
   proc_macro_configs -= [ "//build/config/compiler:chromium_code" ]
   proc_macro_configs += [ "//build/config/compiler:no_chromium_code" ]
+  deps = []
+  if (target_cpu == "arm64") {
+    deps += [ "//brave/third_party/rust/cpufeatures/v0_2:lib" ]
+  }
+  if (target_cpu == "x64" || target_cpu == "x86") {
+    deps += [ "//brave/third_party/rust/cpufeatures/v0_2:lib" ]
+  }
+  if (target_os == "android" && target_cpu == "arm64") {
+    deps -= [ "//brave/third_party/rust/cpufeatures/v0_2:lib" ]
```

Comment:
same as above, I wonder why this is a problem for android if it's not specified in Cargo.toml cc @rillian

Reply:
cc: @cdesouza-chromium

Reply:
As I explained in a previous comment, we had to exclude all dependency to `cpufeatures` from `android`/`arm64`.